gr-blocks/python/blocks/qa_msg_pair_to_var.py

```diff
@@ -36,46 +34,76 @@ def test_f(value: int = 0):
 
         dut = blocks.msg_pair_to_var(test_f)
         test_msg = pmt.cons(pmt.intern("foo"), pmt.from_uint64(canary))
-        src = blocks.message_strobe(test_msg, 10)
+        src = blocks.message_strobe(test_msg, 50)
         self.tb.msg_connect(*(src, pmt.intern("strobe")),
                             *(dut, blocks.msg_pair_to_var.IN_PORT))
+
+        start_time = time.time_ns()
         self.tb.start()
-        time.sleep(15e-3)
+        """
+        sleep for at most a second
+        """
+        time.sleep(50e-3)
+        for counter in range(19):
+            if self.test_val == canary:
+                break
+            time.sleep(50e-3)
```

Comment:
This could be simplified:
```suggestion
        for counter in range(20):
            time.sleep(50e-3)
            if self.test_val == canary:
                break
```

Reply:
(and same for the other test)